### ISSUE TITLE
update golang to 1.11.x?

### ISSUE THREAD

**Issue by jhoblitt** [lawrencegripper/terraform-provider-kubernetes-yaml#5]
There are some smalls differences in `go fmt` between 1.10 and 1.11 -- it would be nice if the entire tf universe would migrate to 1.11 (and module-ize...).

**Comment by lawrencegripper** [comment]
Agreed, for the time being I'm tracking the version of go I wrote the PR against (1.10) but I'd like to upgrade this and move away from using `go-vendor` to using `dep` or the new go 1.11 module stuff (haven't played with it yet)
